aten/src/ATen/mps/MPSStream.mm

```diff
@@ -46,9 +48,13 @@
 }
 
 void MPSStream::commit(bool doFlush) {
+#if USE_MPSCOMMANDBUFFER
+  [commandBuffer() commitAndContinue];
+#else
   if (doFlush) {
```

Comment:
If this is always true, we should remove this condition.

Reply:
Yes, the doFlush is always true.